config/owasp/suppressions.xml

```diff
@@ -24,6 +24,10 @@
     <cve>CVE-2020-5408</cve>
     <cve>CVE-2022-1471</cve>
     <cve>CVE-2022-40152</cve>
+    <cve>CVE-2021-37533</cve>
+    <cve>CVE-2020-8908</cve>
+    <cve>CVE-2018-10237</cve>
+    <cve>CVE-2016-3094</cve>
```

Comment:
Add these CVEs to the <notes> element above just in case we need to add any comments against them such as a Jira ticket reference.